### ISSUE TITLE
Syntax Hightlighting broke on latest version

### ISSUE THREAD

**Issue by MattTheCuber** [mathworks/MATLAB-extension-for-vscode#13]
Just updated to version 1.78.0 and now MATLAB code syntax highlighting doesn't work for some things. Keywords, strings, numbers, brackets, and comments are still highlighted. However, variables and functions are no longer highlighted. I am happy to provide any useful information.

**Comment by dklilley** [comment]
Can you upload an image to show the incorrect highlighting?

Also, can you verify that the current language for the file is "MATLAB"? The language should show in the bottom right corner of VS Code.

**Comment by MattTheCuber** [comment]
![Code_lR9e8tGWsS](https://user-images.githubusercontent.com/32849887/236316284-f3256ed7-7cc6-4591-ac3e-bfe3842d428b.png)

**Comment by dklilley** [comment]
Can you share what the linting warnings states?

I am not able to reproduce the syntax highlighting issue, but I am not seeing the warning on my end.

**Comment by MattTheCuber** [comment]
![image](https://user-images.githubusercontent.com/32849887/236320702-73a46d48-7e65-4c6e-b6c3-c7035bcfc197.png)
The linting error is from the _Matlab Unofficial_ extension. (No, this is not creating the syntax bug, I tried disabling it and restarting VS Code.)

**Comment by hamidingit** [comment]
I am not able reproduce this either. I have 1.78 on windows 11..

**Comment by MattTheCuber** [comment]
Can I provide any logs? Is there a debug version I can use and then provide logs?

**Comment by dklilley** [comment]
Here are a couple ideas:

**Send Text Mate Syntax Grammar Log**
1. Select all the code in your example, and cut
2. Execute the ">Developer: Start Text Mate Syntax Grammar Logging" command - this should open a separate file to display the log.
3. Paste the code back into the file
4. Click the "Stop" button in the popup in the primary VS Code window to stop logging
5. Share this log

**Send Editor Token Info**
1. Ensure the editor has the desired code
2. Execute the ">Developer: Inspect Editor Tokens and Scopes" command
3. Click on a few different words/tokens in the Editor (I would primarily like to see "function", "testFunction", and "end") - this should display a tooltip which shows various information about how each word is being tokenized.
4. Share this information.


Thanks!

**Comment by MattTheCuber** [comment]
[vcode-tm-log-f841f286-5166-4193-8353-152ac74c5951.txt](https://github.com/mathworks/MATLAB-extension-for-vscode/files/11405890/vcode-tm-log-f841f286-5166-4193-8353-152ac74c5951.txt)
![image](https://user-images.githubusercontent.com/32849887/236451739-56633bdf-490d-412d-bfcc-33b10439e1aa.png)
![image](https://user-images.githubusercontent.com/32849887/236451756-5784f62d-0394-4bf5-b0cf-727909681969.png)
![image](https://user-images.githubusercontent.com/32849887/236451782-934a0bc4-2656-4e0c-ac6c-b0efc0a73d2d.png)
![image](https://user-images.githubusercontent.com/32849887/236451804-51bd5708-49e6-41eb-bfde-78dbdcfd3b05.png)
![image](https://user-images.githubusercontent.com/32849887/236451832-a76a90f3-e323-4627-a3ad-b7587cee8c2d.png)
![image](https://user-images.githubusercontent.com/32849887/236451872-71f6aad9-ca78-4306-977f-cebb0262620f.png)

**Comment by dklilley** [comment]
Are you using any particular theme in VS Code? I believe there were some changes/additions to themes in 1.78.0, so I wonder if something may have broken there.

This should be displayed in the `workbench.colorTheme` setting.

**Comment by hamidingit** [comment]
I have found this issue: https://github.com/microsoft/vscode/issues/165225 in VS Code. This could be due to the new color theme which was added in 1.78.

**Comment by MattTheCuber** [comment]
When I updated VS Code it switched to the new "Dark Modern" theme, and I immediately switched to "Dark (Visual Studio)": `"workbench.colorTheme": "Visual Studio Dark",`.

I just tested, and yes syntax highlighting did not work for any other languages.

Changing _Color Theme_ it to "Dark +" has the same style as "Dark (Visual Studio)" and fixed syntax highlighting.
![image](https://user-images.githubusercontent.com/32849887/236467625-65157d15-29a5-4b40-86c5-5efe7ceac4cf.png)
![image](https://user-images.githubusercontent.com/32849887/236468128-b02c4e96-b60b-4df5-887e-94082736a63f.png)

Thank you @dklilley and @hamidingit!
